### DISCUSSION TITLE
Tailwind CLI on raw html php.

### DISCUSSION THREAD

**Discussion by MarinosTBH** [tailwindlabs/tailwindcss#13396] [Help]
I have a project in HTML PHP TAILWIND 
I used CLI to install tailwind
and did this :

```
npx tailwindcss -i ./src/input.css -o ./src/output.css --watch
```
My output.css looks perfect, despite some classes when i add them i neew to restart the command npx tailwind... again.(Not a prob)
The question is: I want to delete the tailwind package after outputting in output.css and deliver the project to my teacher without obliging him to install the npm package to benefit of the tailwindcss ?
Does it work ?

**Comment by mutasim77** [comment]
After generating the `output.css` file, you can use it anywhere without needing tailwind. It's as if someone else wrote the CSS styles for you. As long as you have those class names in your html tags and the `output.css` is linked to your html(or in ur case php) file, everything will work fine.
